README.md

```diff
@@ -13,6 +13,8 @@ General:
 Other:
 * For Open vSwtich (OVS) bridging, the `openvswitch` kernel module
   must loaded on the host system (where docker engine is running).
+* For patch connections (`bridge: patch`), the kernel must support
+  tc qdisc mirred filtering via the `sch_mirred` kernel module.
```

Comment:
This is my ignorance/unfamiliarity, but it seems like `sch_mirred` and `act_mirred` could be loaded independently from each other?  It seems like `act_mirred` supports all the functionality we're using currently (which you noted in the commit message), and it's the one we're actually looking for with `kmod-loaded?`.  Double checking that documenting `sch_mirred` here is the right thing.

Reply:
Ugh, `sch_mirred` was an earlier failed attempt until I realized it was rally `act_mirred` I wanted. Somehow the older name slipped through. I pushed an update. Thanks for doing a thorough review and keeping me honest!